app/server/database/entities/Company.ts

```diff
@@ -0,0 +1,12 @@
+import { Column, Entity } from 'typeorm'
+import { z } from 'zod'
+import { Base } from './Base'
+
+@Entity()
+export class Company extends Base {
+  @Column('text', { unique: true })
```

Comment:
please do not add a ynique constraint here (unless you properly handle it in the endpoint and the frontend): many people may make similar or the same suggestions, and they should eithert all go through or the user should get good feedbackl like: "thanks, but this usggestion was already made!"

I would for now just allow all suggestions to keep the scope of this re-write on point. we can think about filtering out duplicates later on

Reply:
Here, I made 2 entities.
- Company (for saving mlms list)
- Suggestion (for saving suggestions)

I thought as people can make non-make sense suggestions, adding suggestions directly to `Company` could be a bit dangerous. (though suggestion endpoint is not used in frontend yet)
What do you think about this?

For now, I removed the unique constraint from `Suggestion` and left it to Company.